docs/API/Logging.md

```diff
@@ -39,93 +39,98 @@ These advantages are mainly for BetonQuest, but it is also very useful for 3rd p
         !!! abstract ""
-            Simply create a BetonQuestLogger instance in your class as such:
-    
-            === "Without topic"
-                ````java linenums="1" title="Automatically determine the logging class"
-                public final class MyCustomEvent {
-                    private final static BetonQuestLogger LOG = BetonQuestLogger.create();
-                ````
-                ````java linenums="1" title="Manually specify the logging class"
-                public final class MyCustomEvent {
-                    private final static BetonQuestLogger LOG = BetonQuestLogger.create(MyCustomEvent.class);
-                ````
+            This is the recommended way to obtain a BetonQuestLoggerFactory instance. Simply add this in your `onEnable()` method:
             
-            === "With topic"
-                Automatically determine the class for the logger.
-                ````java linenums="1"
-                public final class MyCustomEvent {
-                    private final static BetonQuestLogger LOG = BetonQuestLogger.create("MyCustomTopic");
-                ````
-                Manually specify the class for the logger.
-                ````java linenums="1"
-                public final class MyCustomEvent {
-                    private final static BetonQuestLogger LOG = BetonQuestLogger.create(MyCustomEvent.class, "MyCustomTopic");
-                ````
-    === "Using Lombok"
-        **This is for BetonQuest addons that rely on BetonQuest and have Lombok set up**  
-        Using Lombok enables you to use the handy [@CustomLog](https://projectlombok.org/features/log)
-        annotation on each class you want a logger for.
-        This requires a Lombok setup in your project and in your IDE.
-    
-        !!! abstract "1. Setup"
-            The first step is to install a Lombok plugin in your IDE. IntelliJ contains it by default.
-    
-            All 3rd party plugins need to create a new file named `lombok.config` in their projects root.
-            Copy the following to the file:
-            ````linenums="1"
-            lombok.log.custom.declaration = org.betonquest.betonquest.api.BetonQuestLogger org.betonquest.betonquest.api.BetonQuestLogger.create(TYPE)(TYPE,TOPIC)
-            lombok.log.fieldName = LOG
+            ````java linenums="1"
+            final BetonQuestLoggerFactory loggerFactory = Bukkit.getServicesManager().load(BetonQuestLoggerFactory.class);
+            ````
+            
+            This only works if BetonQuest is installed on the server. If you want to support servers without BetonQuest,
+            you need to do it this way:
+            
+            ````java linenums="1"
+            RegisteredServiceProvider<BetonQuestLoggerFactory> provider = Bukkit.getServicesManager().getRegistration(BetonQuestLoggerFactory.class);
+            if (provider != null) {
+                BetonQuestLoggerFactory loggerFactory = provider.getProvider();
+            }
```

Comment:
This example will fail with a ClassNotFoundException, as the ClassLoader will try to load `BetonQuestLoggerFactory` which does not exist either.

Also, it is not required, as `.getServicesManager().load(BetonQuestLoggerFactory.class)` will just return null if there is no instance provided. But that can only happen if the plugin loads before BetonQuest.